etl/steps/grapher/faostat/2022-05-17/shared.py

```diff
@@ -43,9 +42,6 @@ def get_grapher_dataset_from_file_name(file_path: str) -> catalog.Dataset:
     # Load latest garden dataset.
     dataset = catalog.Dataset(garden_data_dir)
 
-    # Adapt dataset metadata to grapher requirements.
-    dataset.metadata = gh.adapt_dataset_metadata_for_grapher(dataset.metadata)
```

Comment:
I'm a bit confused by this case. Wouldn't the new dataset have no metadata if we remove this line?

Reply:
It loads the metadata of garden dataset on the line above
```
dataset = catalog.Dataset(garden_data_dir)
```
this PR changes makes `grapher://` run `adapt_dataset_metadata_for_grapher` by default for all steps right before upserting to database.

(I've double checked this with `compare --show-values grapher faostat 2022-05-17 faostat_fs --data-values`)

Reply:
Ah fantastic, thank you!